perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AuthzResolverBlImpl.java

```diff
@@ -669,7 +669,7 @@ public static void setRole(PerunSession sess, User user, Role role, List<PerunBe
 	 *
 	 * @param sess                perun session
 	 * @param user                the user for setting role
-	 * @param role                role of user in a session ( perunadmin | voadmin | groupadmin | self | facilityadmin | voobserver | topgroupcreator | securityadmin )
+	 * @param role                role of user in a session ( perunadmin | voadmin | groupadmin | self | facilityadmin | voobserver | topgroupcreator | securityadmin | resourceselfservice )
```

Comment:
Can you please add also "resourceadmin" role to the javadoc. Its not related to your changes, but is missing in all setRole()/unsetRole() methods in this. Thank you.

Reply:
Done.